### PR TITLE
New task JS Variables

### PR DESCRIPTION
**Pull request by Allatest**

### CODE REVIEW

File: js.variables/variables.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+//var userName = "Black";
+
+//console.log(typeof adminName); // string
+//console.log(typeof userName);  // string
+
+//var message = "awesome"
+
+//var password1 = 1234;
+//var password2 = 1111;
+//console.log(`Javascript is: ${message}, ${password1}, ${password2}`);
```
Comment (landriiuk) [inline js.variables/variables.js]: Why is this code commented?

**Review by landriiuk — CHANGES_REQUESTED** [review]

**Comment by landriiuk** [comment]
1. Pls delete node_modules manually, you do have not to push it to git. If somehow changes are not blocked to add to git use full root to node_modules, f.e: /your folder/node_modules/**
2. Ensure that node_modules is not added to your commit
3. Uncomment your code
4. Push new changes to git

File: js.variables/variables.js (hunk 1 above)
Comment (Allatest) [reply js.variables/variables.js]: Have already uncommented my code. Can I merge?

File: js.variables/variables.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+console.log(typeof adminName); // string
+console.log(typeof userName);  // string
+
+var message = "awesome"
+
+var password1 = 1234;
+var password2 = 1111;
+console.log(`Javascript is: ${message}, ${password1}, ${password2}`);
+
+var Mynumber = "23";
```
Comment (landriiuk) [inline js.variables/variables.js:13]: for naming better to use camelCase "myNumber"
```suggestion
var myNumber = "23";
```

File: js.variables/.DS_Store
Comment (landriiuk) [inline js.variables/.DS_Store:1]: Pls delete this file, its auto-generated, and there no need to add it to git

File: js.variables/package-lock.json
Comment (landriiuk) [inline js.variables/package-lock.json:1]: better move this file to the root of your projects. 
Now it's located in js.variables folder better to move it to the root.

File: js.variables/package.json
Comment (landriiuk) [inline js.variables/package.json:1]: better move this file to the root of your projects. 
Now its located in the js.variables folder, better to move it to the root.

File: js.variables/variables.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+var adminName = "Kate";
+var userName = "Black";
```
Comment (landriiuk) [inline js.variables/variables.js:2]: Better to use let or const.

**Comment by landriiuk** [comment]
Pls fix the points I mentioned. Then merge this pr into main

**Pull request merged**